b2share/modules/communities/alembic/fb99eeaec4ac_create_communities_tables.py

```diff
@@ -0,0 +1,44 @@
+"""Create communities tables.
+
+Revision ID: fb99eeaec4ac
+Revises: ec77f5a034d1
+Create Date: 2017-04-25 17:45:13.538252
+
+"""
+import uuid
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.sql import expression
+from sqlalchemy.types import TIMESTAMP
+from sqlalchemy_utils.types import UUIDType
+
+
+# revision identifiers, used by Alembic.
+revision = 'fb99eeaec4ac'
+down_revision = 'ec77f5a034d1'
+branch_labels = ()
+depends_on = None
```

Comment:
Can you add a comment just to point developers to the module in which this other alembic script is defined.
ex:
```
depends_on = 'dbdbc1b19cf2' # in invenio-base
```